.packit.yaml

```diff
@@ -14,198 +14,142 @@ srpm_build_deps:
   - hatch
   - python3-hatch-vcs
 
-jobs:
+actions: &base-actions
+  create-archive:
+    - hatch run docs:man
+    - hatch build -t sdist
+    - bash -c "ls dist/tmt-*.tar.gz"
+  get-current-version:
+    - hatch version
 
-  # Build pull requests
-  - job: copr_build
+jobs:
+  - &copr_build
+    job: copr_build
     trigger: pull_request
-    targets:
+    targets: &all-targets
```

Comment:
@LecrisUT let's put this in the section after actions definition, so the common pieces are on one place rather?

Reply:
You mean top-level one? Never tried to do that, let me check if it works.

Reply:
yeah, that is what I meant, but we can keep it also this way, not sure if it works

Reply:
I have since, did it in a couple of projects. works just fine, I have the change stashed